schemas/pipeline.go

```diff
@@ -0,0 +1,187 @@
+package schemas
+
+import (
+	"fmt"
+)
+
+// Pipeline represents an errand pipeline which consists of errands and dependencies between them.
+// Cyclic dependencies are not allowed.
+//easyjson:json
+type Pipeline struct {
+	Name              string `json:"name" binding:"required"`
+	DeleteOnCompleted bool   `json:"deleteOnCompleted,omitempty"`
+
+	Errands      []Errand             `json:"errands,omitempty" binding:"required"`
+	Dependencies []PipelineDependency `json:"dependencies,omitempty"`
+
+	// Attributes added by errands server
+	ID            string `json:"id"`
+	Status        string `json:"status,omitempty"`
+
+	// TODO: use ptime once that's public
```

Comment:
i believe it is

Reply:
It's still private, seems like it's importing some internal libs (go-lib-apis, go-lib-account-services-models)